### DISCUSSION TITLE
docker 部署python3 manage.py makemigrations sql 报错

### DISCUSSION THREAD

**Discussion by chenjinlei1122** [hhyo/Archery#2508] [Q&A]
### 重现步骤

![image](https://github.com/hhyo/Archery/assets/32697166/d5a6651d-ebed-4f2f-bf78-c5ad6ee9da58)
![image](https://github.com/hhyo/Archery/assets/32697166/dd52612f-9fbf-485b-a29a-7e3b526e133c)
mysql, redis配置如上，同步命令出错

### 预期外的结果

出现报错

### 版本

最新master

### 部署方式

Docker

**Comment by chenjinlei1122** [comment]
有大佬帮忙吗

**Reply by chenjinlei1122** [reply]
@LeoQuote 辛苦大佬看一下

**Comment by LeoQuote** [comment]
这边无法复现, 帮不了

**Reply by chenjinlei1122** [reply]
我只改了这两个配置，然后.env里去掉了这两配置，执行就报错了，是不是要重启一下docker服务

**Reply by LeoQuote** [reply]
试试呗, 我这边确实帮不了哈, 都没法复现的

**Reply by chenjinlei1122** [reply]
重启docker 服务也不行
